### DISCUSSION TITLE
ERC721A

### DISCUSSION THREAD

**Discussion by alexkalen** [smartcontractkit/full-blockchain-solidity-course-py#1710] [Q&A]
Hey guys, can anyone with a little more experience using brownie tell me how to use Azuris's ERC721A contract as a base using brownie?

**Comment by cromewar** [comment]
Hello @alexkalen 
Sorry for the delay, well you can just use it as any other package like OpenZeppelin and Chainlink.
just import it as dependency on `brownie-config.yaml`, something like this:

```YAML
dependencies:
   - chiru-labs/ERC721A@4.2.3
```

The github for ERC721A is the following: https://github.com/chiru-labs/ERC721A

I hope this info might help.
Cheers.
